pithos/plugin.py

```diff
@@ -18,16 +18,23 @@
 import os
 from gi.repository import (
     GLib,
-    Gio
+    Gio,
+    GObject
 )
 
 
-class PithosPlugin:
+class PithosPlugin(GObject.Object):
+    __gtype_name__ = 'PithosPlugin'
+    __gsignals__ = {
```

Comment:
Why make it a signal rather than a property, you still get change notifications but you get the benefit of it being a toggle and can be bound.